tests/src/test/scala/cats/tests/WriterTTests.scala

```diff
@@ -16,7 +16,7 @@ class WriterTTests extends CatsSuite {
   // Scalacheck to calm down a bit so we don't hit memory and test duration
   // issues.
   implicit override val generatorDrivenConfig: PropertyCheckConfiguration =
-    PropertyCheckConfiguration(minSuccessful = 20, sizeRange = 5)
+    checkConfiguration.copy(sizeRange = 5)
```

Comment:
Keeping `minSuccessful` as it is set in `CatsSuite` only seems to add around half a second to the total test time of `WriterTTests`.